install/configuration/settings.py

```diff
@@ -0,0 +1,235 @@
+import os, sys
+
+# Build paths inside the project like this: os.path.join(BASE_DIR, ...)
+BASE_DIR = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+PROJECT_ROOT = os.path.dirname(__file__)
+
+# SECURITY WARNING: keep the secret key used in production secret!
+SECRET_KEY = ''
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
+
+# DOMAINS AND URLS
+SERVER_DOMAIN = ''
+ALLOWED_HOSTS = ['*']
+LOGIN_URL = '/login/'
+
+# Internationalization
+LANGUAGE_CODE = 'en-us'
+TIME_ZONE = 'UTC'
+DATETIME_FORMAT = 'Y-m-d H:i:s e'
+USE_I18N = True
+USE_L10N = False
+USE_TZ = True
+
+# MISC
+CRISPY_TEMPLATE_PACK = 'bootstrap3'
+ROOT_URLCONF = 'app.urls'
+WSGI_APPLICATION = 'app.wsgi.application'
+
+# STATIC FILES
+STATIC_URL = '/static/'
+STATIC_ROOT = os.path.join(BASE_DIR, "staticfiles")
+STATICFILES_DIRS = [
+    os.path.join(BASE_DIR, "static")
+]
+
+# DATABASE
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+        'NAME': '',
+        'USER': os.environ.get('AA_DB_DEFAULT_USER', ''),
+        'PASSWORD': os.environ.get('AA_DB_DEFAULT_PASSWORD', ''),
+        'HOST': os.environ.get('AA_DB_DEFAULT_HOST', '127.0.0.1'),
```

Comment:
With docker the mysql service will join the network under name `db` (I think), may need to test this and try it out here instead of `127.0.0.1`

Reply:
I just confirmed that the container name for the database specified in the docker-compose.yml file works as a hostname from within the auth docker container. Pretty nifty.